py/desispec/fiberbitmasking.py

```diff
@@ -79,7 +79,12 @@ def get_fiberbitmasked_frame_arrays(frame,bitmask=None,ivar_framemask=True,retur
         if bitmask.isnumeric():
             bad = np.int32(bitmask)
         else:
-            bad = get_fiberbitmask_comparison_value(kind=bitmask)
+            band = 'b' # default, which is the most constraining (for stdstar fit)
```

Comment:
Perhaps we could do band = 'brz' that masks if any amp is set. Otherwise you're default will miss actually flagged fibers if the true camera is r or z.
Note this suggestion breaks your later logic of only checking the first character of "band" in `get_all_fiberbitmask_with_amp(band)`

Reply:
yes, I am replacing 'b' by 'brz' and I am changing the function get_all_fiberbitmask_with_amp to test all bands